manifests/init.pp

```diff
@@ -50,8 +46,12 @@
 #
 # $unattended_url::               URL hosts will retrieve templates from during build (normally http as many installers don't support https)
 #
-# $authentication::               Enable user authentication. Initial credentials are set using admin_username
-#                                 and admin_password.
+# $locations_enabled::            Enable locations? (Unused since 1.21)
+#
+# $organizations_enabled::        Enable organizations? (Unused since 1.21)
+#
+# $authentication::               Enable user authentication. (Unused since 1.21)
+#                                 Initial credentials are set using admin_username and admin_password.
```

Comment:
Looks like there was also an oversight here: these are now prefixed with `initial_`.

Reply:
fixed on the rebase